index.html

```diff
@@ -61,7 +61,7 @@
 
   <section id="getting-started">
     <aside>
-    <h3>Alright. How do I get started?</h3>
```

Comment:
I would argue that this is a use of "Alright" with a slightly different meaning to "All right". Similar to "already" and "all ready", I don't think they are entirely interchangeable.

But I'm with Ward that maybe just avoiding the issue is preferable. "OK" would be a fine substitute :+1: